src/stan-users-guide/efficiency-tuning.qmd

```diff
@@ -1366,102 +1386,83 @@ data {
   vector[N] x;
 }
 transformed data {
+  real mean_x = mean(x);
+  real sd_x = sd(x);
   vector[N] x_std;
-  vector[N] y_std;
-  x_std = (x - mean(x)) / sd(x);
-  y_std = (y - mean(y)) / sd(y);
+  x_std = (x - mean_x) / sd_x;
 }
 parameters {
   real alpha_std;
   real beta_std;
-  real<lower=0> sigma_std;
+  real<lower=0> sigma;
 }
 model {
   alpha_std ~ normal(0, 10);
   beta_std ~ normal(0, 10);
-  sigma_std ~ cauchy(0, 5);
-  for (n in 1:N) {
-    y_std[n] ~ normal(alpha_std + beta_std * x_std[n],
-                      sigma_std);
-  }
+  sigma ~ cauchy(0, 5);
+  y ~ normal(alpha_std + beta_std * x_std, sigma);
 }
 ```
 
 The parameters are renamed to indicate that they aren't the
 "natural" parameters, but the model is otherwise identical.  In
-particular, the fairly diffuse priors on the coefficients and error
-scale are the same.  These could have been transformed as well, but
+particular, the fairly diffuse priors on the coefficients are the same.
+These could have been transformed as well, but
 here they are left as is, because the scales make sense as
-diffuse priors for standardized data; the priors could be made more
-informative.  For instance, because the outputs $y$ have been
-standardized, the error $\sigma$ should not be greater than 1, because
-that's the scale of the noise for predictors $\alpha = \beta = 0$.
+diffuse priors for standardized data.
 
 The original regression
 $$
 y_n
 = \alpha + \beta x_n + \epsilon_n
 $$
 has been transformed to a regression on the standardized variables,
+
 $$
-\textrm{z}_y(y_n)
-= \alpha'
-+ \beta' \textrm{z}_x(x_n)
-+ \epsilon'_n.
+y_n
+= \alpha' + \beta' \textrm{z}_x(x_n) + \epsilon_n.
 $$
-The original parameters can be recovered with a little algebra,
-\begin{align*}
-y_n &= \textrm{z}_y^{-1}(\textrm{z}_y(y_n)) \\
-    &= \textrm{z}_y^{-1}
-\left( \alpha' + \beta' \textrm{z}_x(x_n) + \epsilon_n' \right) \\
-    &= \textrm{z}_y^{-1}
-\left( \alpha' + \beta' \left( \frac{x_n - \bar{x}}{\texttt{sd}(x)} \right) + \epsilon_n' \right) \\
-    &= \texttt{sd}(y)
-\left( \alpha' + \beta' \left( \frac{x_n - \bar{x}}{\texttt{sd}(x)} \right) + \epsilon_n' \right) + \bar{y} \\
-    &=
-\left( \texttt{sd}(y) \left( \alpha' - \beta' \frac{\bar{x}}{\texttt{sd}(x)} \right) + \bar{y} \right)
-+ \left( \beta' \frac{\texttt{sd}(y)}{\texttt{sd}(x)} \right) x_n
-+ \texttt{sd}(y) \epsilon'_n,
-\end{align*}
+
 from which the original scale parameter values can be read off,
-$$
+
+
+\begin{align*}
 \alpha
-=
-\texttt{sd}(y)
-      \left(
-          \alpha'
-          - \beta' \frac{\bar{x}}{\texttt{sd}(x)}
-      \right)
-  + \bar{y};
-\qquad
-\beta = \beta' \frac{\texttt{sd}(y)}{\texttt{sd}(x)};
+= \alpha'
+-
+\beta' \frac{\bar{x}}{\texttt{sd}(x)};
 \qquad
-\sigma = \texttt{sd}(y) \sigma'.
+\beta = \frac{\beta'}{\texttt{sd}(x)};
+\end{align*}
+
+The independent priors on alpha and beta map to a multivariate normal prior on alpha_std and beta_std. 
+
+$$
+Cov(\alpha, \beta) = 0 \implies
```

Comment:
You can't just write "Cov" in math mode---it'll be a mess of ill-spaced italics.  What you want to do is \text{Cov}.  Then, you need to be careful because it's a function defined for random variables.  

What if you start with an informative joint prior?

Reply:
removed this addition.